plugin/src/autoworkletization.ts

```diff
@@ -124,6 +125,9 @@ function processArguments(
       // workletizable argument doesn't always have to be specified
       return;
     }
+    if (maybeWorklet.isReferencedIdentifier() && maybeWorklet.isIdentifier()) {
```

Comment:
We need this condition in such shape since JSX tags can be referenced identifiers but they're not identifiers.